plugins/modules/vmware_vc_infraprofile_info.py

```diff
@@ -0,0 +1,238 @@
+author:
+- Naveenkumar G P (@ngp)
+notes:
+- Tested on vSphere 6.5, 6.7
+requirements:
+- python >= 2.6
+- PyVmomi
+- vSphere Automation SDK
+options:
+    decryption_key:
+      description:
+      - decryption_key argument for while doing import profle task as of now its not taken into account form API team.
+      type: str
+      required: False
+    encryption_key:
+      description:
+      - encryption_key argument for while doing import profle task as of now its not taken into account form API team.
+      type: str
+      required: False
+    api:
+      description:
+      - which api need to be executed(list,export,import,validate)
+      type: str
+      required: False
+      choices: [ list, export, import, validate ]
+    config_path:
+      description:
+      - config file path which contains infra profile config json data, supports both realtive and absolute path.
+      - this parameter is required only when import,validate APIs are being used.
+      type: str
+      required: False
+    profiles:
+      description:
+      - list of profile name to be exported, imported, and validated.
+      - this parameter is not required while running for List API, not for export,import and validate.
+      type: str
+      required: False
+    description:
+      description:
+      - description about encryption or decrytption key
+      type: str
+      required: False
+extends_documentation_fragment: 
+- community.vmware.vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Create Content Library
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+  delegate_to: localhost
+
+- name: export Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+  delegate_to: localhost
+
+- name: validate Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "validate"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+
+- name: import Vcenter appliance infra profile config
+  vmware_vc_infraprofile_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    validate_certs: False
+    validate_certs: False
+    api: "export"
+    profiles: "ApplianceManagement"
+    config_path: "export.json"
+  delegate_to: localhost
+  '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.vmware.plugins.module_utils.vmware_rest_client import VmwareRestClient
+import json
+import os
```

Comment:
Unused import

Reply:
@Akasurde 
shall remove these imports?

Reply:
Yes.

Reply:
if I remove import compilation will not happen right, because the rest of the modules are having imports on top?